modules/js/src/embindgen.py

```diff
@@ -155,9 +103,16 @@ def makeWhiteList(module_list):
                 wl[k] = m[k]
     return wl
 
-white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, calib3d])
+track = ""
+if "OPENCV_JS_WHITELIST" in os.environ:
+    track = os.environ["OPENCV_JS_WHITELIST"]
+else:
+    track = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'opencv_js.config.py')
+with open(track, 'r') as config_file:
+     config_data = config_file.read()
+white_list = None
+exec(compile(config_data, track, 'exec'))
```

Comment:
Please use just
```python
exec(open(track).read())
```
I think that it's better to have different name for a variable. In example, `white_list_config`